### PR TITLE
Bump masmovil_bazel_rules

### PR DESCRIPTION
**Pull request by lukas-slezevicius**
## What is the goal of this PR?

Bump masmovil_bazel_rules and stop depending on Vaticle's fork. The PRs submitted by Vaticle were merged into masmovil_bazel_rules, therefore we don't need to maintain the fork anymore.

## What are the changes implemented in this PR?

* Bump masmovil_bazel_rules and stop depending on Vaticle's fork

### CODE REVIEW

**Comment by grabl** [comment]
### PR Review Checklist

Do not edit the content of this comment. The PR reviewer should simply update this comment by ticking each review item below, as they get completed.

----
**Code**

- [ ] Packages, classes, and methods have a single domain of responsibility.
- [ ] Packages, classes, and methods are grouped into cohesive and consistent domain model.
- [ ] The code is canonical and the minimum required to achieve the goal.
- [ ] Modules, libraries, and APIs are easy to use, robust (foolproof and not errorprone), and tested.
- [ ] Logic and naming has clear narrative that communicates the accurate intent and responsibility of each module (e.g. method, class, etc.).
- [ ] The code is algorithmically efficient and scalable for the whole application.

**Architecture**

- [ ] Any required refactoring is completed, and the architecture does not introduce technical debt incidentally.
- [ ] Any required build and release automations are updated and/or implemented.
- [ ] Any new components follows a consistent style with respect to the pre-existing codebase.
- [ ] The architecture intuitively reflects the application domain, and is easy to understand.
- [ ] The architecture has a well-defined hierarchy of encapsulated components.
- [ ] The architecture is extensible and scalable.

File: distribution/helm/deps.bzl [hunk 1]
```diff
@@ -1,9 +1,8 @@
-load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
+load("@bazel_tools//tools/build_defs/repo:git.bzl", "git_repository")
 
 def deps():
-    http_archive(
+    git_repository(
```
Comment (lolski) [inline distribution/helm/deps.bzl:4]: Let's tweak the ordering: declare the `name`, `remote` and then `commit`.

Comment (lukas-slezevicius) [reply distribution/helm/deps.bzl:4]: done.

Comment (lolski) [reply distribution/helm/deps.bzl:4]: Oh and also remove the `.git` suffix for the `remote` value. Our convention is to not add them.

You should look at other `deps.bzl` files in other packages to learn more about our coding convention.

**Comment by lukas-slezevicius** [comment]
The files created by this bump are inconsistent with previous fork. Need to investigate why.

**Pull request closed without merging**